genesis-tools/genesis-csv-to-json/src/main.rs

```diff
@@ -18,9 +20,18 @@ fn main() {
                 .takes_value(true),
         )
         .arg(Arg::with_name("chain-id").long("chain-id").takes_value(true))
+        .arg(Arg::with_name("tracked-shards").long("tracked-shards").takes_value(true))
         .get_matches();
 
     let home_dir = matches.value_of("home").map(|dir| Path::new(dir)).unwrap();
     let chain_id = matches.value_of("chain-id").expect("Chain id is requried");
-    csv_to_json_configs::csv_to_json_configs(home_dir, chain_id.to_string());
+    let tracked_shards: HashSet<ShardId> = match matches.value_of("tracked-shards") {
```

Comment:
How is this arg used? The usage of this command line tool is to allow a group of participants who share the same csv file to create the same genesis config. Are they all going to track the same set of shards? Will this set of shards include all shards existing, if not who is going to track the remaining shards?

Reply:
Please add explanation to the arg's description.

Reply:
Tracked shards are independent of genesis. They are specified by the node if the node wants to always track some shards. I'll add an explanation.

Reply:
Could you also answer:
> Are they all going to track the same set of shards? Will this set of shards include all shards existing, if not who is going to track the remaining shards?

Reply:
And add this explanation to the arg description.

Reply:
No. As I said tracked-shards is specified by the node. If a node wants to track some shards, it is their decision. It is not related in any way to how any other nodes make this decision.

Reply:
How do people decide what shards to track? Let's say we have stake wars participants, how does participant Alice decide that she wants to track some set of shards, e.g. "1,5,6"? How do they know what is the total number of shards?